idaes/apps/uncertainty_propagation/examples/BT_NRTL_dataset.csv

```diff
@@ -0,0 +1,51 @@
+temperature,liq_benzene,vap_benzene

```

Comment:
Suggestion. Add README.md in this directory and explicitly state where this data came from. Check the IDAES NRTL example for the correction citation for the dataset.

Reply:
I can find [tutorials](BT_NRTL_dataset.csv) using `BT_NRTL_dataset.csv` but don't know the reference. I asked Jaffer. I will update if I can check it.